apps/mull-ui/src/app/components/profile-header/profile-header.tsx

```diff
@@ -0,0 +1,88 @@
+import { faShareAlt } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { MullButton } from '..';
+import './profile-header.scss';
+
+function checkFriendStatus(isFriend: boolean) {
+  let friendStatus: string;
+  if (isFriend === true) {
+    friendStatus = 'Friends';
+  }
+  if (isFriend === false) {
+    friendStatus = 'Pending';
+  }
+  // No relation in the Friends table
+  if (isFriend === null) {
+    friendStatus = 'Add Friend';
+  }
+  return (
+    <div className="friend-status-container">
+      <MullButton className={isFriend ? 'friend-status-button' : 'not-friend-status-button'}>
+        {friendStatus}
+      </MullButton>
+    </div>
+  );
+}
+
+export interface profileHeaderProps {
+  userName?: string;
+  userPicture?: string;
+  userPortfolio?: number;
+  userFriends?: number;
+  userHosting?: number;
+  userDescription?: string;
+}
+
+export const ProfileHeader = ({
+  userName = '',
+  userPicture = '',
+  userPortfolio = 0,
+  userFriends = 0,
+  userHosting = 0,
+  userDescription = '',
+}: profileHeaderProps) => {
+  // TODO: Check UserID, if currentUserID == profileUserID, then it is true, so return without the Friend button, otherwise return with the Friend button
+  const isCurrentUser = true;
+  const isFriend = true;
+  return (
+    <div className="profile-background">
```

Comment:
```suggestion
    <div className="profile-header-container">
```
I think this makes more sense this div. Let me know what you think.